README.md

```diff
@@ -11,202 +11,273 @@ The goal of the project is to create a common HTML5 UI abstraction layer for the
 
-## Bindings
+## Prerequisites
+
+Your compiler must support minimum C++11 except for platforms that require a more modern version.
+
+### Linux
+
+The [GTK][gtk] and [WebKit2GTK][webkitgtk] libraries are required for development and distribution. You need to check your Linux distribution regarding how to install those those.
+
+Debian-based systems:
+
+* Packages:
+  * Development: `apt install libgtk-3-dev libwebkit2gtk-4.0-dev`
+  * Production: `apt install libgtk-3-0 libwebkit2gtk-4.0-37`
 
-This repository contains bindings for C, C++, and Go. Bindings for other languages are maintained separately.
+BSD-based systems:
 
-* [Rust](https://github.com/Boscop/webview-rs)
-* [Python](https://github.com/zserge/webview-python)
-* [Nim](https://github.com/oskca/webview)
-* [Haskell](https://github.com/lettier/webviewhs)
-* [C#](https://github.com/webview/webview_csharp)
-* [Janet](https://github.com/janet-lang/webview)
-* [Crystal](https://github.com/naqvis/webview)
-* [Java](https://github.com/shannah/webviewjar)
-* [Ruby](https://github.com/Maaarcocr/webview_ruby)
-* [Pascal](http://github.com/PierceNg/fpwebview)
+* FreeBSD packages: `pkg install webkit2-gtk3`
+* Execution on BSD-based systems may require adding the `wxallowed` option (see [mount(8)](https://man.openbsd.org/mount.8))  to your fstab to bypass [W^X](https://en.wikipedia.org/wiki/W%5EX "write xor execute") memory protection for your executable. Please see if it works without disabling this security feature first.
 
-Instructions for [Go](#webview-for-go-developers) and [C/C++](#webview-for-cc-developers) are included below.
+### Windows
 
-## Webview for Go developers
+Your compiler must support C++17 and we recommend to pair it with an up-to-date Windows 10 SDK.
 
-### Getting started
+For Visual C++ we recommend Visual Studio 2022 or later. We have a [separate section for MinGW-w64](#mingw-w64-requirements).
 
-Install this library with `go get`:
+Developers and end-users must have the [WebView2 runtime][ms-webview2-rt] installed on their system for any version of Windows before Windows 11.
+
+## Getting Started
+
+If you are a developer of this project then please go to the [development section](#development).
+
+To keep this section simple, instructions here are written for GCC when compiling C/C++ code using Unix-style command lines, and assumes that you run multiple commands in the same shell. See the [MinGW-w64 requirements](#mingw-w64-requirements) when building on Windows.
+
+Onward we are going to show you just enough to get a working app but you are encouraged to explore the [available examples](examples) and try the ones that go beyond the mere basics.
+
+Start with creating a new directory structure for your project because you will need it to follow along.
 
```

Comment:
The directions here are wordy. I think there might be too much commentary like "To keep this section simple" and "Onward we are going to show you" and "because you will need it to follow along".

Reply:
I've removed some of the non-essential parts.